tensorboard/compat/tensorflow_stub/io/gfile_test.py

```diff
@@ -192,6 +199,28 @@ def testReadWithOffset(self):
             ckpt_read = f.read()
             self.assertEqual(ckpt_b_content, ckpt_read)
 
+    def testWrite(self):
```

Comment:
There should be at least one test that checks the behavior of multiple writes to a single GFile instance.  Ideally also things like an empty write and a write that overwrites an existing file.  And ideally also for the S3 implementation.